### DISCUSSION TITLE
Artwork repository started

### DISCUSSION THREAD

**Discussion by pphotography** [endless-sky/endless-sky#9325] [Show and tell]
Recently I started experimenting with AI image creation. I found 'Stable Diffusion' which is open-source and runs locally on your computer. After experimenting with its setting via the trial-and-error method I was able to create several planet and station artwork images for Endless Sky in 720x360 resolution.
 
![20230922_19-49-25_screenshot](https://github.com/endless-sky/endless-sky/assets/49001045/4d2fc22b-18d5-4683-8ba2-62a5d87d2f88)

I categorized and uploaded several images to a Github repository. Initially it contains:
 

- CityStreets: Views of the everyday life in megacities with people moving around the streets
- ClearSky: These are planets with a clear and clean athmosphere. Many images could be from the same civilization. 
 This could be a highly sophisicated civilization that spread out on many planets and is using terraforming to create the environment they want
 - ColoredCivilization: These civilizations could have enourmous increase in population. They managed to spread out on several planets but they are not much technologically sophisticated because they must manage more and more inhabitants
 
- Dark: This is a set of dark and mostly scary planets.
 
- Desert: Some installations on desert planets
 
- Dust: These planets are near to stars and not much habitable. No breathable atmosphere, high temperature and many sandstorms make life on these planets not much comfortable
 
-  Fantasy: Many civilizations on these planets did not evolve to space travel or don't want to. They live a medieval live but guests from outer space are welcome to chat and make business with them.
 
- Greenlands: These civilizations are technologically high advanced, they managed to integrate technology and society in a environmental-friendly way. They enjoy their comfortable lives and the nature around them.
 
- HardWeather: Many planets in space are not suitable for supporting life on them. No plants or lifeforms have evolved here. The athmosphere is not breathable; clouds, rain and heavy winds. But these planets have resources, therefore many are inhabited. The installations there are not made to look good, they just have to withstand the unfriendly environment

- Industrial: These civilizations are stuck on a very early industrial age. They produce cheap but they ruin their planets with their exhaustion gas and chemical waste. They don't repair or invent much. If a production site collapses they just build another one. If a planet gets inhabitable they just move to the next one to ruin it

- Interior: These are interiors of space ships or stations

- Landscapes: These are uninhabited planets. Maybe I will divide them into subfolders in the future (ice, desert, fantasy etc.).

- Lava: So hot that even the ground is melting. Only a few are inhabited.

- Megacities: These civilizations have evolved so high that most of their population live in megacities. Many images look similar as if they are from the same civilization that spread out on several planets

- Moons: These are moons and planets without or a very thin athmosphere. On the surface you can view directy on stars or other planets. There are many mining or sciencs stations on them

- Night: These are some planets at night, many images could be from the same civilization

- SingleStations: On these planets single installations are in focus (Radars, Beacons, Communication relays, outposts etc.)

- Snow: Winter and snow is a phenomena that is not only common on earth, therefore some planets are covered in snow. 

- SpacePorts: In Endless Sky we are visiting space ports on many planets to get the latest news and sometimes missions. But what do they look like? Here are some ideas. Maybe you can integrate images for space ports into the game?

- Valley: These planets have many rocks and valley on their surface. They are mostly used for bigger mining operations

- WaterWorlds: There are many planets in space where their surface is mostly covered by water. People live in installations under the sea or on the water surface

You can find the repo here:
https://github.com/pphotography/EndlessSky_Artwork

**Comment by Saugia** [comment]
Unfortunately, we are not accepting AI art for Endless Sky. This has been a discussion brought up often, but the overall stance is between being limited on Steam due to AI asset usage (generally Steam would ban the use of it), copyright grounds and licensing is very grey/not easily workable, and community members that would much prefer to create the assets themselves rather than have a program do so for them, AI art isn't accepted. 

You may use it for plug-ins or the likes though (at your own discretion), so you may display this and some creators may be interested, but we will not be using it in vanilla Endless Sky.

**Comment by pphotography** [comment]
Thanks for your information. I found out that making a plugin is quite easy. When I place an image with a filename from the original game into the plugins folder images/land/ the game will use it. Savegames are not affected when I remove the plugin again.
Currently I start with making substitutes for the original images in this plugin based on the appearance of the original files. When I am finished I will make a new repository for this plugin. If someone wants to contribute to this every help is welcome since there are 456 images to replace
